test/integration/scheduler_perf/scheduler_bench_test.go

```diff
@@ -45,20 +45,23 @@ var (
 	// From PV controller
 	annBindCompleted = "pv.kubernetes.io/bind-completed"
 
-	tests = []struct{ nodes, existingPods, minPods int }{
+	defaultTests = []struct{ nodes, existingPods, minPods int }{
 		{nodes: 500, existingPods: 500, minPods: 1000},
 		{nodes: 5000, existingPods: 5000, minPods: 1000},
 	}
+	testNamespace  = "sched-test"
```

Comment:
Can we use a single namespace?

Reply:
We could, but some features work at the namespace level, so I find it useful to test that.